netlify/functions/repositories/ReviewRepository.ts

```diff
@@ -123,6 +123,88 @@ class ReviewRepository {
       .where("id", "=", id)
       .execute();
   }
+
+  async getReviewsByWorkId(clubId: string, workId: string) {
+    return db
+      .selectFrom("work_list")
+      .where("work_list.club_id", "=", clubId)
+      .where("work_list.type", "=", WorkListType.reviews)
+      .innerJoin("work_list_item", "work_list_item.list_id", "work_list.id")
+      .innerJoin("work", "work.id", "work_list_item.work_id")
+      .where("work.id", "=", workId)
+      .leftJoin("review", "review.work_id", "work.id")
+      .leftJoin(
```

Comment:
Do we need all these joins for the work details? Does the endpoint get these in a separate call to the database?